roles/testnode/tasks/ntp.yml

```diff
@@ -3,7 +3,7 @@
   yum:
     name: ntp
     state: present
-  when: ansible_pkg_mgr  == "yum"
+  when: ansible_pkg_mgr  == "yum" or ansible_pkg_mgr == "dnf"
```

Comment:
Just change to `ansible_os_family == "RedHat"`

Reply:
I ended up dropping the commit